src/quickstatements_client/model.py

```diff
@@ -169,8 +169,8 @@ class BaseLine(BaseModel):
 
     subject: str = Field(regex=r"^(LAST)|(Q\d+)$")
     predicate: str = Field(
-        regex=r"^(P\d+)|(Len)|(Den)$",
-        description="Either a predicate, `Len` (label), or `Den` (description)",
+        regex=r"^(P\d+)|([ADE]{1}[a-z]{2})$",
```

Comment:
I had a really hard time figuring out how quickstatements worked in the beginning and I want to make sure that the package explicitly explains how things work. Therefore, I would ask if you want to extend this, then you write everything out in full, with code comments. `[ADE]{1}[a-z]{2}` unfortunately doesn't mean anything, and the changes to the description string serve to obfuscate what's going on here, so I can't accept these changes as is

One way to do that would be to create a list or dictionary data structure with all of these parts in it, then assemble the regex programatically.

Reply:
by "dictionary data structure" you mean a dict of valid two-letter language codes?

Reply:
Ohhh I see what's going on here. The `en` at the end of `Len` is a two letter code. As far as I can remember, I did not see this anywhere, and it didn't even occur to me. So it can be `L{two letter code}` for label, `D{two letter code}` for description, and something else for Alts?  How did you get to ADE?

Maybe do something like this

```python
two_letter_codes = [
    "en",
    # fill in, or get programatically somehow
]

regex_part = "|".join(two_letter_codes)
regex = rf"^(P\d+)|([ADL]({regex_part}))$"
```

Reply:
Source:  https://www.wikidata.org/w/index.php?title=Help:QuickStatements&section=6#Adding_labels,_aliases,_descriptions_and_sitelinks

I'll work out something along your suggestion, thanks.

the "E" is a mistake, should be "L". That's fixed in a subsequent commit.